eslint-rules/lib/rules/prop-value-shape-deprecation.js

```diff
@@ -54,7 +54,10 @@ module.exports = {
     function reportPropValueShapeDeprecation(propKey, prop, deprecation, node) {
       const componentName = utils.getComponentName(node);
       const newProp = _.get(deprecation, 'fix.propName');
-      const message = `The shape of '${prop}' prop of '${componentName}' doesn't contain '${deprecation.prop}' anymore. Please use '${newProp}' instead (fix is available).`;
```

Comment:
OK, sorry, I was wrong about adding a fix message (not about the possibility of no fix), but the message should (partially) coming from the `deprecation.message` (see the components' rule for an example).

Reply:
Nice! Thanks and sorry again.